<?xml version="1.0"?>

        <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
        <!--                                                                                    -->
        <!--                                                                                    -->
        <!--                            LAUNCH SENSOR PLACEMENTS MAP                            -->
        <!--                                                                                    -->
        <!--                                                                                    -->
        <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
        <!--                              @AsifArain, Dec 08, 2017                              -->
        <!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->

<launch>

	<node pkg="plan_execution" type="sensor_placements_map_publisher_node" name="placements_map_publisher" output="screen">
	
	    <param name="map_file"	        value="prismaforum4_map_conf.dat"/>
        <param name="map_size_file"	    value="prismaforum4_mapsize_conf.dat"/>
        <param name="cell_size_file"	value="prismaforum4_cellsize_conf.dat"/>
        <param name="robot_origin_file"	value="prismaforum4_origin_conf.dat"/>
        
        <param name="hard_offset_x"	    value="-0.5"/> <!--"0.0"-->
        <param name="hard_offset_y"	    value="-0.25"/> <!--"-0.25"-->
        
        
    </node>
</launch>


